python/lsst/cp/pipe/deferredCharge.py

```diff
@@ -229,6 +229,11 @@ def solveLocalOffsets(self, inputMeasurements, calib, detector):
         calib : `lsst.ip.isr.DeferredChargeCalib`
             Populated calibration.
 
+        Raises
+        ------
+        RuntimeError
+            Raised if no data remains after flux filtering.
+
         Notes
         -----
         The original CTISIM code uses a data model in which the
```

Comment:
Wher's the "original" CTSIM code? Should there be a reference, for readers not familiar with the context?